CODEOWNERS

```diff
@@ -193,19 +179,75 @@ extensions/filters/http/oauth2 @rgs1 @derekargueta @snowp
 
+# "Unowned extensions (without 1 maintainer or 2 owners)
+# DNS resolution
+/*/extensions/network/dns_resolver/cares @yanavlasov @UNOWNED
+/*/extensions/network/dns_resolver/apple @yanavlasov @UNOWNED
+
+/*/extensions/formatter/req_without_query @UNOWNED @tsaarni
+# Stateful session
+/*/extensions/http/stateful_session/cookie @wbpcode @UNOWNED
+/*/extensions/filters/http/stateful_session @wbpcode @UNOWNED
+# ext_authz
+/*/extensions/filters/common/ext_authz @esmet @gsagula @UNOWNED
+/*/extensions/filters/http/ext_authz @esmet @gsagula @UNOWNED
+/*/extensions/filters/network/ext_authz @esmet @gsagula @UNOWNED
+# compression code
+/*/extensions/filters/http/decompressor @rojkov @UNOWNED
+/*/extensions/filters/http/gzip @gsagula @UNOWNED
+/*/extensions/filters/http/common/compressor @gsagula @rojkov @UNOWNED
+/*/extensions/filters/http/compressor @rojkov @UNOWNED
+/*/extensions/compression/brotli @rojkov @UNOWNED
+/*/extensions/compression/common @UNOWNED @rojkov
+/*/extensions/compression/gzip/decompressor @rojkov @UNOWNED
+/*/extensions/compression/gzip/compressor @rojkov @UNOWNED
+/*/extensions/compression/gzip/common @rojkov @UNOWNED
+# cel
+/*/extensions/access_loggers/filters/cel @UNOWNED @douglas-reid
+# rbac
+/*/extensions/filters/network/rbac @UNOWNED @UNOWNED
+/*/extensions/filters/http/rbac @UNOWNED @UNOWNED
+/*/extensions/filters/common/rbac @UNOWNED @UNOWNED
+# tap
+/*/extensions/filters/http/tap @mattklein123 @UNOWNED
+/*/extensions/common/tap @mattklein123 @UNOWNED
+/*/extensions/transport_sockets/tap @mattklein123 @UNOWNED
+/*/extensions/filters/network/local_ratelimit @mattklein123 @UNOWNED
+/*/extensions/filters/http/ip_tagging @UNOWNED @UNOWNED
+/*/extensions/filters/http/health_check @UNOWNED @UNOWNED
+/*/extensions/filters/http/cors @UNOWNED @UNOWNED
+/*/extensions/filters/http/dynamo @UNOWNED @UNOWNED
+/*/extensions/filters/http/lua @UNOWNED @UNOWNED
+/*/extensions/filters/http/common @UNOWNED @UNOWNED
+/*/extensions/filters/common @UNOWNED @UNOWNED
+/*/extensions/filters/common/lua @UNOWNED @UNOWNED
+/*/extensions/filters/listener/original_dst @UNOWNED @UNOWNED
+/*/extensions/filters/listener/proxy_protocol @UNOWNED @UNOWNED
+/*/extensions/stat_sinks/statsd @UNOWNED @UNOWNED
+/*/extensions/stat_sinks/common @UNOWNED @UNOWNED
+/*/extensions/stat_sinks/common/statsd @UNOWNED @UNOWNED
+/*/extensions/access_loggers/grpc @UNOWNED @UNOWNED
+/*/extensions/access_loggers/file @UNOWNED @UNOWNED
+/*/extensions/tracers/zipkin @UNOWNED @UNOWNED
+/*/extensions/tracers/dynamic_ot @UNOWNED @UNOWNED
+/*/extensions/tracers/opencensus @UNOWNED @UNOWNED
+/*/extensions/tracers/lightstep @UNOWNED @UNOWNED
+/*/extensions/tracers/common @UNOWNED @UNOWNED
+/*/extensions/tracers/common/ot @UNOWNED @UNOWNED
+/*/extensions/retry/host/previous_hosts @UNOWNED @UNOWNED
+/*/extensions/filters/network/client_ssl_auth @UNOWNED @UNOWNED
+/*/extensions/filters/network/echo @UNOWNED @UNOWNED
+/*/extensions/filters/network/mongo_proxy @UNOWNED @UNOWNED
```

Comment:
I think move to contrib at this point.

Reply:
I think move this to contrib. Lyft doesn't use this anymore and I don't know if anyone else does.

Reply:
We still use this at Stripe, I'd be happy to help to take a first pass as I help maintain our internal forks part of this code.

Reply:
Hi, I can help with mongo_proxy.

Reply:
Hi @Mythra if Stripe could help own this (and also upstream fork patches!) that would be fantastic. We can keep this in core if people are using it. cc @alyssawilk

Reply:
(Just to make sure the loop is closed) Yeah we'd be happy too.